purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -105,7 +105,7 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
     private val dispatcher: Dispatcher,
     private val identityManager: IdentityManager,
     private val subscriberAttributesManager: SubscriberAttributesManager,
-    @JvmSynthetic internal var appConfig: AppConfig
+    @get:JvmSynthetic internal var appConfig: AppConfig
```

Comment:
should this one be `@set:JvmSynthetic @get:JvmSynthetic` since it's a `var` and not `val`?